consensus/src/experimental/README.md

```diff
@@ -0,0 +1,122 @@
+splits the bottleneck into smaller parts. To tolerate execution uncertainties like bit flips, DiemBFT let the
+nodes agree on the execution result. The nodes asynchronously vote on the execution result. Note that, this
+extra roundtrip is not piggybacked on the consensus communication (HotStuff chains), therefore, it does not
+adversely affect the high throughput and low latency. To help slower nodes catch up, DiemBFT nodes also
+share aggregated signatures on executed results.
+
+# Diagram
+
+Recall that in the DiemBFT protocol, every block goes through a number of notorizations before become decided.
+Fortunately, the blocks form a pipeline of chain rule. 
+On average, each round (that is not a timeout) commits a new block.
+Therefore, the throughput can be roughly estimated as 1/(the time per round). 
+![Chain rule in DiemBFT](docs/chain.png)
+If we look closer to the implementation (shown above), each step consists of executing the block, voting the block, 
+and (after a while) committing (persisting) the block. All of these steps are on the critical path.
+![Chain rule in DiemBFT](docs/before.png)
+Therefore, the throughput is at most 1/(the time of preparation + the time of execution + the time of voting + the time of committing/persisting).
+
+Decoupled execution moves execution and persisting to dedicated threads/tasks. The time cost of execution and committing/persisting
+are no longer on the critical path of the main loop.
+![Chain rule in DiemBFT](docs/after.png)
+By Amdahl's rule, this roughly changes the throughput to 
+1/min(the time of proposing + the time of voting, the time of execution, the time of committing/persisting),
+which, if ignoring the synchronization cost between threads, is presumably better than the previous throughput.
+We also decoupled the signing phase (not shown on the figure) from the main loop.
+
+# Evaluation Results
+
+By turning on decoupled execution feature, the throughput roughly improves 90% from the cluster benchmark.
+
+# Implementation Details
+
+    
+            ┌──────────────────┐
+            │ 2. Signing Phase │
+            └──────────────▲─┬─┘
+                           │ │
+    ┌────────────────────┐ │ │ ┌─────────────────────┐
+    │ 1. Execution Phase │ │ │ │ 4. Persisting Phase │
+    └─────────────────▲─┬┘ │ │ └┬─▲──────────────────┘
+                      │ │  │ │  │ │
+        0. Ordered  ┌─┴─▼──┴─▼──▼─┴────┐ 3. Commit Vote  ┌─────────┐
+           Blocks   │                  ├─────────────────►         │
+          ┌─────────►  Buffer Manager  │                 │ Network │
+          │         │                  ◄─────────────────┤         │
+     ┌────┴─────┐   └─────────▲────────┘    Commit Vote  └─────────┘
+     │ Ordering │             │
+     │ State    │   Sync Req  │
+     │ Computer ├─────────────┘
+     └──────────┘
+
+
+Below is a list of changes in a chronological order.
```

Comment:
I don't think we need to provide this type of changelog in the readme, instead we should focus on explaining the responsibility of each component as well as the overall workflow (english version of the above figure